### PR TITLE
feat: Added Event Details Page

### LINKED ISSUE #38: Add Event Details Page
**Describe the feature you'd like to add**
Add Event Details Page

**Would you like to work on the issue?**
- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #38 
Changes: 
Added Event Details page
Added Picasso library to load images.

Screenshots for the change:
Screenshot_20190528-182115

Screenshot_20190528-182120

### CODE REVIEW

**Comment by amdal07** [comment]
@somenath1435 please squash the commits. Its tough to review

**Comment by somenath1435** [comment]
@amdal07 please review

File: app/src/main/java/com/example/saar/EventAdapter.java [hunk 1]
```diff
@@ -44,10 +51,24 @@ public TimeLineViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(TimeLineViewHolder holder, int position) {
+    public void onBindViewHolder(final TimeLineViewHolder holder, final int position) {
```
Comment (amdal07) [inline app/src/main/java/com/example/saar/EventAdapter.java]: why final?

File: app/src/main/res/layout/event_list_row.xml [hunk 2]
```diff
@@ -7,7 +7,8 @@
     android:clickable="true"
     android:focusable="true"
     android:orientation="horizontal"
-    xmlns:tools="http://schemas.android.com/tools">
```
Comment (amdal07) [inline app/src/main/res/layout/event_list_row.xml]: unwanted changes

File: app/src/main/java/com/example/saar/EventAdapter.java (hunk 1 above)
Comment (somenath1435) [reply app/src/main/java/com/example/saar/EventAdapter.java]: If not final, then we cannot use holder and position inside onclick()

**Pull request merged**